core/build.gradle

```diff
@@ -179,8 +193,8 @@ shadowJar {
   zip64 = true
   // avoid conflict with the 0.9.2 version in stock Spark
   relocate 'org.apache.thrift', 'io.snappydata.org.apache.thrift'
-  // relocate fastutil for possible conflicts with user dependencies
-  relocate 'it.unimi.dsi.fastutil', 'io.snappydata.it.unimi.dsi.fastutil'
+  // relocate koloboke for possible conflicts with user dependencies
+  relocate 'com.koloboke', 'io.snappydata.com.koloboke'
```

Comment:
This is only for the snappydata-core shadow jar. That one is used for smart connector, so the shadow jar that is added has relocated package name as well as all callers are updated. This avoids any potential clash with other jars that user may be having in application.

For embedded mode, there is no relocation. These are loaded by the Spark cluster boot themselves and thus have precedence for all usages withing the Spark/SnappyData product code itself. In case of jobs using separate koloboke version, those will be loaded in their own ClassLoader so will not clash with product jars.